README.md

```diff
@@ -2,15 +2,25 @@
 
 [![Build Status](https://travis-ci.org/deis/router.svg?branch=master)](https://travis-ci.org/deis/router) [![Go Report Card](http://goreportcard.com/badge/deis/router)](http://goreportcard.com/report/deis/router)
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, [Heroku-inspired](http://heroku.com) workflow.
+Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, easy and secure way to deploy your code to production.
 
-The router component, specifically, handles ingress and routing of HTTP/S traffic bound for the Deis API and for your own applications.  This component is 100% Kubernetes native and is useful even without the rest of Deis!
+## Beta Status
 
-## Work in Progress
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please submit an [issue][issues]. If you'd like to participate in development, please read the "Development" section below and submit a [pull request][prs].
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+# About
 
-Deis Router v2 is changing quickly. Your feedback and participation are more than welcome, but be aware that this project is considered a work in progress.
+The Deis router handles ingress and routing of HTTP/S traffic bound for the Deis API and for your own applications. This component is 100% Kubernetes native and, while it's intended for use inside of the Deis [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service), it's flexible enough to be used as a standalone pod on any Kubernetes cluster.
```

Comment:
> it's flexible enough to be used as a standalone pod on any Kubernetes cluster

Not quite accurate... but only because you said "pod."  I suppose it's a small nuance of the router, that it _must_ be run with an RC.  The annotations that drive the configuration of the router are expected to appear on the RC, not on the pod...

This is a tangent, but I would have _preferred_ that each router instance just read the annotations of its own pod.  These, of course, would have been specified by the RC's template spec for the pods.  The problem is that if that changes, the changes aren't propagated to the running pods.  Since it's vitally important that router be able to re-configure itself without downtime, all router instances look for annotations on the RC itself for configuration.  No RC == no configuration.

I don't think there's any need to go into this level of detail here, especially given that the charts get this right _and_ the documentation is clear that the annotations belong on the RCs.  I think if we just lose the "standalone pod" language, we'll be covered.